doc/adoc/user_guide.adoc

```diff
@@ -62,9 +62,62 @@ To install the distribution migration system run:
 [listing]
 tux > sudo zypper in SLE15-Migration suse-migration-sle15-activation
 
+== Customize the Migration Process
+
+The migration system reads a custom config file from the system to upgrade.
+The contents of that file allows to influence the behavior of the upgrade
+process. Prior to the start of the upgrade process, create the following
+file:
+
+[listing]
+tux > ssh INSTANCE_USER@IP_OF_INSTANCE 'touch /etc/sle-migration-service.yml'
+
+The custom config file allows for the following settings:
+
+Enable Debug Mode::
+If enabled, prevents the upgrade system from rewinding the setup
+steps and rebooting due to a failed upgrade, allowing the issue to
+be debugged.
+
+[listing]
+debug: true|false
+
+Specify Migration Product::
+With the SLE15-Migration package a predefined product setup is
+already applied. The value for the target product can be overwritten
+with this setting. Following product names can be set:
+
+* SLES/15/x86_64
+* SLES/15.1/x86_64
+
+[listing]
+migration_product: SLES/15/x86_64
+
+[NOTE]
+If the migration plugin is not used, the value set for the target
```

Comment:
At this point the reader knows nothing about plugins and cannot relate the information presented to any other concept previously discussed. If we talk about "plugins" here we need to set the context earlier in the document to describe the difference between plain "zypper dup" and "zypper migration".

The other option is to re-write this in a way that simply addresses, in a brief way the differences between "zypper dup" and "zypper migration" and when might want to use one vs. the other. Then state that this setting allows the user to switch between the two methods.

Reply:
I tried that now, hope it's more self explaining now